### PR TITLE
Adjust egress proxy resources and replicas

### PR DESCRIPTION
**Pull request by ludydoo**
Bumped the memory of egress proxy
Reduced from 3 to 2 replicas

### CODE REVIEW

File: fleetshard/pkg/central/charts/data/tenant-resources/values.yaml [hunk 1]
```diff
@@ -1,12 +1,11 @@
 egressProxy:
   image: ubuntu/squid:5.2-22.04_beta
-  replicas: 3
```
Comment (stehessel) [inline fleetshard/pkg/central/charts/data/tenant-resources/values.yaml:3]: Why the change from 3 to 2? Does it matter that we'll have less throughput performance with only 2 replicas?

Comment (ludydoo) [reply fleetshard/pkg/central/charts/data/tenant-resources/values.yaml:3]: The 3 replicas are overkill. They are idling 99% of the time and just consuming memory:

Screenshot 2023-12-07 at 2 55 29 PM
Screenshot 2023-12-07 at 2 56 14 PM
